tests/viscoplastic_melt_fraction.prm

```diff
@@ -0,0 +1,121 @@
+# instead of seconds' parameter is set; seconds otherwise.
+set End time                               = 1000
+set Pressure normalization                 = surface
+set Surface pressure                       = 0
+set Resume computation                     = false
+set Start time                             = 0
+set Use years in output instead of seconds = true
+set Output directory                       = output-vp-melt-fraction
+
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = top, bottom
+  set Model name = initial temperature
+
+  subsection Initial temperature
+    set Maximal temperature = 2773
+    set Minimal temperature = 273
+  end
+end
+
+subsection Boundary composition model
+  set List of model names = initial composition
+end
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent  = 500e3
+    set Y extent  = 500e3
+  end
+end
+
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    # Magnitude of the gravity vector in $m/s^2$. The direction is always
+    # radially outward from the center of the earth.
+    set Magnitude = 1e-4
+  end
+end
+
+subsection Compositional fields
+  set Number of fields = 2
+  set Names of fields = porosity, bound_fluid
+end
+
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Function expression = 0; 0.01
+  end
+end
+
+# The composition is not fixed on any boundary
+subsection Boundary composition model
+  set Fixed composition boundary indicators =
+  set List of model names = initial composition
+end
+
+subsection Initial temperature model
+  set Model name = function
+
+  subsection Function
+    set Function constants  = bw = 500e3, Tmax = 2773, Tmin = 273
+    set Function expression = x*(Tmax-Tmin)/bw+Tmin
+    set Variable names      = x,y
+  end
+end
+
+# Use a visco plastic material model, which uses the dislocation
+# and diffusion creep mechanisms.
+subsection Material model
+  set Model name =  reactive fluid transport
+  set Material averaging =  harmonic average
+  subsection Reactive Fluid Transport Model
+    set Base model = visco plastic
+    set Fluid-solid reaction scheme = katz2003
+  end
+
+end
+
+subsection Mesh refinement
+  set Coarsening fraction                      = 0.05
+  set Refinement fraction                      = 0.3
+  set Initial adaptive refinement              = 0
+  set Initial global refinement                = 5
+  set Strategy                                 = composition
+  set Time steps between mesh refinement       = 0
+end
+
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = 0,1,2,3
+  set Zero velocity boundary indicators       =
+end
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Use direct solver for Stokes system = true
+  end
+end
+
+subsection Postprocess
+  set List of postprocessors = visualization, melt statistics
```

Comment:
Can you add composition statistics here? To test if the reactions work, one of the properties we care about is the porosity compositional field (which should be zero at the start, and then larger, after the reactions have occurred).

Reply:
For comprehensiveness: @jdannberg and I had a conversation offline and decided that the reactive transport functionality will be added in a separate PR. A new test will be made then that tracks composition specifics to test if reactions work.